styles/styles.css

```diff
@@ -0,0 +1,348 @@
+body {
+  margin: 0;
+  width: 100%;
+}
+
+/*
+header {
+  background-color: #0A3C61;
+  height: 21rem;
+  text-align: left;
+  display: flex;
+  justify-content: center;
+}
+*/
+header {
+  background-color: #0A3C61;
+  height: 24rem;
+  justify-content: flex-start;
+
+}
+
+header h1 {
+  color: black;
+  position: absolute;
+  left: 13rem;
+  top: 1rem;
+  font-size: 60px;
+  font-family: sans-serif;
+}
+
+header h2 {
+  color: white;
+  position: absolute;
+  font-size: 25px;
+  font-family: sans-serif;
+  font-style: italic;
+  line-height: 36px;
+}
+
+.b-box {
+  width: 30rem;
+  height: 100px;

```

Comment:
make sure you're consitent with px/rem on heights. Below you use REM's here you use px.

Either is fine, but I prefer REMS

Reply:
replaced with rem